newIDE/app/src/EventsSheet/ParameterFields/ImageResourceField.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+import { Trans } from '@lingui/macro';
+
+import * as React from 'react';
+import ResourceSelector from '../../ResourcesList/ResourceSelector';
+import ResourcesLoader from '../../ResourcesLoader';
+import {
+  type ParameterFieldProps,
+  type ParameterFieldInterface,
+} from './ParameterFieldCommons';
+
+const ImageResourceField = React.forwardRef<
```

Comment:
Any chance you can add it to a story? :)

Reply:
Yes, will do!